tests/Processor/DateProcessorTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+
+namespace RenanBr\BibTexParser\Test\Processor;
+
+
+use DateTime;
+use DateTimeInterface;
+use DateTimeZone;
+use PHPUnit\Framework\TestCase;
+use RenanBr\BibTexParser\Processor\DateProcessor;
+
+class DateProcessorTest extends TestCase
+{
+    public function testDateYearAndMonth()
+    {
+        $processor = new DateProcessor();
+        $entry = $processor([
+            'month' => '"1~" # jan',
```

Comment:
It should test against `1~jan`, because of [this listener's behaviour](https://github.com/renanbr/bibtex-parser/blob/b1922ca1654ce7fffe5f9648099e06ef17994f9b/src/Listener.php#L81-L98)

Reply:
We need to check it against `1~jan`